api/test/dummy.test.js

```diff
@@ -0,0 +1,3 @@
+describe('Dummy test', () => {
+  test('Dummy test', () => {});
```

Comment:
why the dummy test

Reply:
Added it in there to make the tests pass for deployment but I saw you fixed this with `yarn test --passWithNoTests`, thanks!